.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,21 @@
+name: Feature request
+description: "Let us know what you have in mind"
```

```suggestion
description: "Suggest a feature for PostHog"
```
Let's keep the description from the old template